tests/integration/test_rocksdb_options/test.py

```diff
@@ -30,6 +30,24 @@ def test_valid_options(start_cluster):
     DROP TABLE test;
     """
     )
+    node.query(
+        """
+    CREATE TABLE test (key UInt64, value String) Engine=EmbeddedRocksDB(-1) PRIMARY KEY(key);
+    DROP TABLE test;
+    """
+    )
+    node.query(
+        """
+    CREATE TABLE test (key UInt64, value String) Engine=EmbeddedRocksDB(-1, true) PRIMARY KEY(key);
+    DROP TABLE test;
```

Comment:
you tested only if tables are created not the functionality itself

Reply:
yep, but I could not find any other functionality test of embededrocksdb.

Reply:
Do I miss it? Or shall we open a new dictory and add functionality test of embededrocksdb.

Reply:
check `tests/queries/0_stateless`